### DISCUSSION TITLE
Mis task no se visualizan en channel view

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#764] [2. Questions and Answers]
Hola!

**Descripción:**
Estoy trabajando en mi preset de mantenimiento y ocurre algo curioso. Como contexto: tengo un mantenimiento correctivo que crea tareas según ciertos criterios analizados a través de un faas. Cada vez que creo estas tareas ya sea como plan o orden de trabajo en diferentes workflows, estas no se visualizan en el channel View. En algunas task incluso estoy como editor.
No puedo entender bien por qué sucede esto, o en que me debería fijar para que no suceda. 

Y si accedo a la task view, veo todas las tareas.

**Cómo debería comportarse**
Debería visualizar las task tanto en la channel como en la task View.

**Ambiente**
Staging

**Pasos para repetir**
Ingresar al workflow de plan preventivos y ejecutar a través del required survey el faas. 
Observar las task a través de channel y task view.

**Imagenes de referencia**

Workflow de Órdenes de Trabajo

Channel view:
<img width="1423" alt="Captura de Pantalla 2022-03-15 a la(s) 19 37 36" src="https://user-images.githubusercontent.com/42095617/158483861-17976b94-9338-402e-b9f7-4839902929c0.png">

Task view:
<img width="1386" alt="Captura de Pantalla 2022-03-15 a la(s) 19 37 50" src="https://user-images.githubusercontent.com/42095617/158483905-8eb58e7c-39c1-4a45-8883-5e3e6b5058f3.png">

Sucede lo mismo en el workflow de planes preventivos

**Comment by acedward** [comment]
Esto puede ser efecto de dos cosas : Opciones de visualización de tareas por permisos: sólo se ven en vista de tarea. (ó era efecto secundario del bug de recreación de presets que fue corregido) 
Pronto tendremos mejoras significativas en la vista de tareas, así poder usarlo como vista por defecto.
